Py/DICOMSegmentationPlugin.py

```diff
@@ -143,54 +143,67 @@ def load(self,loadable):
       (success,labelNode) = slicer.util.loadLabelVolume(labelFileName, returnNode=True)
 
       # Initialize color and terminology from .info file
-      # Format of the .info file:
+      # See SEG2NRRD.cxx for how it's written.
+      # Format of the .info file (no leading spaces, modifiers are optionally written if present):

```

Comment:
Not only modifiers, but complete attributes may be missing. It probably makes sense to set Category=Tissue Type=Tissue of none is present. Anatomic Region is optional. Category and Type are mandated by the standard, but we should not crash if the object is non-compliant.